web/docs/project/custom-vite-config.md

```diff
@@ -111,3 +111,58 @@ If you, for example, want to serve the client from a different path than `/`, yo
     ```
   </TabItem>
 </Tabs>
+
+## Chrome DevTools automatic workspace mapping {#devtools-workspace}
+
+Want edits you make in Chrome DevTools Sources panel to save directly to your project files? You can enable [Chrome DevTools Workspace](https://developer.chrome.com/docs/devtools/workspace/) support in development with the community plugin **vite-plugin-devtools-json** (no Wasp fork required).
```

Comment:
Rewrote it a bit. We tell people want they can accomplish, and what they'll have to do to accomplish it. This way docs are super nice to read! I also added a link to the plugin's page.

```suggestion
Chrome DevTools support [mapping a page's resources to a folder](https://developer.chrome.com/docs/devtools/workspaces), so any changes you make in the browser reflect back to your files. To enable it, you can add their Vite plugin: [`vite-plugin-devtools-json`](https://github.com/ChromeDevTools/vite-plugin-devtools-json).
```